include/pmacc/PMaccConfig.cmake

```diff
@@ -343,8 +343,8 @@ if(ALPAKA_ACC_GPU_CUDA_ENABLE)
                             "(Found ${CUDA_VERSION})")
     endif()
     # Newer CUDA releases: probably troublesome, warn at least
-    if(CUDA_VERSION VERSION_GREATER 11.1)
-        message(WARNING "Untested CUDA release >11.1 (Found ${CUDA_VERSION})! "
+    if(CUDA_VERSION VERSION_GREATER 11.2)
+        message(WARNING "Untested CUDA release >11.12 (Found ${CUDA_VERSION})! "
```

Comment:
Why does message have 11.12 and not 11.2?

Reply:
typo